.env.example

```diff
@@ -0,0 +1,6 @@
+# See docs/development/releasing.md
+
+APPVEYOR_TOKEN=
+CIRCLE_TOKEN=
+ELECTRON_GITHUB_TOKEN=
+VSTS_TOKEN=
```

Comment:
The other non-S3 environment variable in my notes is `CHANGELOG_GITHUB_TOKEN` but it wasn't clear to me if it's required for releasing -- another question for @jkleinsc

Reply:
CHANGELOG_GITHUB_TOKEN isn't needed.  That was something I was using to test an alternate changelog generator.